engine/src/main/java/org/hibernate/validator/internal/properties/javabean/JavaBeanGetter.java

```diff
@@ -88,6 +76,11 @@ public String getParameterName(ExecutableParameterNameProvider parameterNameProv
 		return declaringClass;
 	}
 
+	@Override
+	public ConstrainedElementKind getConstrainedElementKind() {
```

Comment:
It seems that `JavaBeanMethod` already returns the same element kind. Do we need to override it here?

Reply:
Fixed.